sphinxcontrib/video.py

```diff
@@ -148,6 +153,27 @@ def run(self) -> List[video_node]:
         ]
 
 
+class VideoPostTransform(SphinxPostTransform):
+    """Ensure video files are copied to build directory.
+
+    This copies what is done for images in the post_process_image method of
+    sphinx.builders.Builder, except as a Transform since we can't hook into that method
+    directly.
+    """
+
+    default_priority = 200
```

Comment:
how did you choose the value of the default priority ? Are you positive it cannot clash with other vanilla SphinxPostTransform object ?

Reply:
To be honest, I don't quite remember. Looking at [docutils](https://docutils.sourceforge.io/docs/ref/transforms.html#transform-priority-range-categories), a 200 puts it in the "very early" stage.

It is a lower value / higher priority than most of Sphinx's builtin transforms, except for [`ReferencesResolver`](https://github.com/sphinx-doc/sphinx/blob/ae51974e217cc7590e1b9c68f18af3e043027c6d/sphinx/transforms/post_transforms/__init__.py#L61), [`OnlyNodeTransform`](https://github.com/sphinx-doc/sphinx/blob/ae51974e217cc7590e1b9c68f18af3e043027c6d/sphinx/transforms/post_transforms/__init__.py#L226), [`ImageDownloader`](https://github.com/sphinx-doc/sphinx/blob/ae51974e217cc7590e1b9c68f18af3e043027c6d/sphinx/transforms/post_transforms/images.py#L47), [`CitationReferenceTransform`](https://github.com/sphinx-doc/sphinx/blob/ae51974e217cc7590e1b9c68f18af3e043027c6d/sphinx/builders/latex/transforms.py#L535), [`MathReferenceTransform`](https://github.com/sphinx-doc/sphinx/blob/ae51974e217cc7590e1b9c68f18af3e043027c6d/sphinx/builders/latex/transforms.py#L556) and [`ViewCodeAnchorTransform`](https://github.com/sphinx-doc/sphinx/blob/ae51974e217cc7590e1b9c68f18af3e043027c6d/sphinx/ext/viewcode.py#L189). It is equal priority to [`SigElementTransformFallback`](https://github.com/sphinx-doc/sphinx/blob/ae51974e217cc7590e1b9c68f18af3e043027c6d/sphinx/transforms/post_transforms/__init__.py#L239), and [`PropagateDescDomain`](https://github.com/sphinx-doc/sphinx/blob/ae51974e217cc7590e1b9c68f18af3e043027c6d/sphinx/transforms/post_transforms/__init__.py#L282).

I think it makes sense to be higher value / lower priority to `OnlyNodeTransform`, and possibly `ImageTransform`. I don't _think_ the rest matter?